.github/workflows/api_tests.yml

```diff
@@ -11,9 +11,11 @@ name: API Tests
 
 on:
   push:
-    branches: [main]
+    branches:
+      - main
   pull_request:
-    branches: [main]
+    branches:
+      - "**"
```

Comment:
Note that '**' is the default and is required to match branches with `/` in the name per https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-syntax#filter-pattern-cheat-sheet